plugins/binance/binance_cli.go

```diff
@@ -0,0 +1,25 @@
+package binance
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func BinanceCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Binance CLI",
+		Runs:    []string{"binance-cli"},
+		DocsURL: sdk.URL("https://github.com/binance/binance-cli"),
+		NeedsAuth: needsauth.IfAll(
+			needsauth.NotForHelpOrVersion(),
+			needsauth.NotWithoutArgs(),
```

Comment:
I think these commands do not require authentication either: https://github.com/binance/binance-cli#market-data

Reply:
Thanks, updated this as per the suggestion